src/Layer.js

```diff
@@ -295,7 +295,6 @@ export default class Layer {
         this.map.setLayoutProperty(this.id, 'visibility', 'visible');
         this._visible = true;
         this.requestData();
-        this._fire('updated');
```

Comment:
Why don't we need to raise the event anymore?

Reply:
I think we never needed it, I removed it because, while looking for something else I realized that. The reason is not needed is because the change of visibility is going to trigger a repaint and the repaint is going to trigger the `updated` event